File: magma-platform/magma-platform-bukkit/src/main/java/de/jalumu/magma/platform/bukkit/text/BukkitTextProvider.java
package de.jalumu.magma.platform.bukkit.text;

import de.jalumu.magma.text.Text;
import de.jalumu.magma.text.TextProvider;

public class BukkitTextProvider extends TextProvider {
    @Override
    protected Text text(String text) {
        return new BukkitText().setText(text);
    }
}

File: magma-platform/magma-platform-bukkit/src/main/kotlin/de/jalumu/magma/platform/bukkit/text/BukkitTextProvider.kt
package de.jalumu.magma.platform.bukkit.text

import de.jalumu.magma.text.Text
import de.jalumu.magma.text.TextProvider

/**
 * The bukkit implementation for the Text api
 */
class BukkitTextProvider : TextProvider() {
    override fun text(text: String): Text {
        return BukkitText().setText(text)
    }
}